### PR TITLE
Add second aerosol model and runtime aerosol model selection

### LINKED ISSUE #4: add additional `aerosol_t` extending class

### PR DESCRIPTION
**Pull request by mattldawson**
closes #4

### CODE REVIEW

File: src/model.F90 [hunk 1]
```diff
@@ -33,7 +44,7 @@ subroutine demo_rad()
       do i_time = 1, 5
          ! Initialize the timestep
          call demo_rad_ccpp_physics_timestep_initial('toy_suite',             &
-              errmsg, errcode)
+              aerosol_model_name, errmsg, errcode)
```
Comment (gold2718) [inline src/model.F90:47]: Why do you need `aerosol_model_name` here (along with at run and final times)? Moving this item from the host table to a module table can fix this.

File: src/model.meta [hunk 2]
```diff
@@ -14,6 +14,12 @@
   type = integer
   units = count
   dimensions = ()
+[ aerosol_model_name ]
+  standard_name = aerosol_model_name
+  units = 1
+  type = character
+  kind = len=*
+  dimensions = ()
```
Comment (gold2718) [inline src/model.meta:22]: Moving this to a module takes it out of the call list (not needed except at init time).

**Review by gold2718 — COMMENTED** [review]
I think this example could use some more cleanup to serve as a demonstration of how to create a real version.

File: src/musica_aerosol.meta [hunk 3]
```diff
@@ -5,3 +5,8 @@
 [ccpp-arg-table]
   name  = aerosol_t
   type  = ddt
+
+[ccpp-table-properties]
+  name = aerosol
+  type = scheme
+
```
[the remark this answers is not in the document]
Comment (mattldawson) [reply src/musica_aerosol.meta]: not if you say it's not... removed!

File: src/musica_aerosol_modal.F90 [hunk 4]
```diff
@@ -12,9 +12,6 @@ module musica_aerosol_modal
 
   public :: aerosol_modal_t
 
-  public :: aerosol_modal_init
-  public :: aerosol_modal_run
-
   ! a modal aerosol state and diagnostics
   !> \section arg_table_aerosol_modal_t  Argument Table
   !! \htmlinclude aerosol_modal_t.html
```
[the remark this answers is not in the document]
Comment (mattldawson) [reply src/musica_aerosol_modal.F90]: removed

**Review by gold2718 — APPROVED** [review]
This looks clean now.

**Pull request merged**